packages/blade/src/components/index.ts

```diff
@@ -17,7 +17,7 @@ export * from './Input/TextInput';
 export * from './Input/PasswordInput';
 export * from './Input/TextArea';
 export * from './Input/OTPInput';
-export * from './Input/SelectInput';
+export * from './Input/DropdownInputTriggers';
```

Comment:
Make sure no existing exports are restricted now that there is a change here?

Reply:
This diff is because of directory rename. The index.ts is not really changed so it continues to export the same things it was exporting earlier.